californium-core/src/main/java/org/eclipse/californium/core/network/stack/BlockwiseLayer.java

```diff
@@ -237,6 +240,12 @@ public void onTimeout() {
 				public void onCancel() {
 					clearBlock1Status(key);
 				}
+
+				@Override
+				public void onSendError(Throwable error) {
+					copyToken();
+					clearBlock1Status(key);
+				}
```

Comment:
I toke the implementation from `onReject()`/`onTimeout()`.
Without the `onSendError()` it would be a `onTimeout()`, so it should be OK.